packages/ecs/src/Engine.ts

```diff
@@ -49,7 +57,7 @@ export class Engine {
     const UndefinedEntity = bitECS.addEntity(HyperFlux.store)
   }
 
-  api: FeathersApplication<ServiceTypes>
+  api: FeathersClient
```

Comment:
we shouldn't overload this here, since we access it on the backend and it will have different types

Reply:
we should be able to declare a type override in client-core that will only be applied in packages that import client-core

Reply:
Weird, I made this change because of type errors when I started, but maybe something has changed in dev since then, because now it's not showing any type mismatches when I revert to what it was.